lib/modules/manager/maven-wrapper/extract.ts

```diff
@@ -28,8 +34,8 @@ function extractLineInfo(lines: string[], regex: RegExp): Version | null {
       const match = regex.exec(line);
       if (match?.groups) {
         return {
-          url: match.groups.url,
-          version: match.groups.version,
+          url: match.groups.url ?? null,
+          version: match.groups.version ?? null,
```

Comment:
why this change? looks like it's not required

Reply:
can revert that indeed, 
I tought it was nicer that url if it was null instead of undefined when not there

Reply:
please revert changes.